strawberryfields/compilers/gaussian_unitary.py

```diff
@@ -18,18 +18,62 @@
 from strawberryfields import ops
 from strawberryfields.parameters import par_evaluate
 from thewalrus.symplectic import (
-    expand_vector,
     expand,
     rotation,
     squeezing,
     two_mode_squeezing,
     interferometer,
     beam_splitter,
 )
-
+from numba import jit
 from .compiler import Compiler
 
 
+@jit(nopython=True)
+def _apply_symp_one_mode_gate(S_G, S, r, i):
```

Comment:
```suggestion
def _apply_symp_one_mode_gate(S_G, S, r, i): # pragma: no cover
```

Reply:
hopefully this won't be needed once the coverage is updated with the new tests i wrote!